iib/workers/tasks/opm_operations.py

```diff
@@ -194,7 +194,9 @@ def _serve_cmd_at_port(serve_cmd, cwd, port, max_tries, wait_time):
     raise IIBError(f'Index registry has not been initialized after {max_tries} tries')
 
 
-def _get_or_create_temp_index_db_file(base_dir, from_index=None):
+def _get_or_create_temp_index_db_file(
+    base_dir, from_index=None, overwrite_from_index_token=None, ignore_existing=False
+):
```

Comment:
Can you explain the two cases for ignore_existing? we are using it only at one place with true

Reply:
I did not want to change behavior so I added this parameter - I thought it can be used in other functions but checking it now it does not seems like it.

Do you agree with this or should I change it back?

Reply:
yeah, if we don't see a future scenario where both values would be useful, I think we should stick to one